pkg/vsphere/toolbox/backdoor_test.go

```diff
@@ -30,13 +30,8 @@ func TestBackdoorChannel(t *testing.T) {
 	}
 
 	for _, f := range funcs {
-		err := f()
-
-		if err != nil {
-			if err == ErrNotVirtualWorld {
```

Comment:
The check will fail if the process isn't run as root so the error returned can be that or the not a virtual world error. I tried getting the tests to run as root in drone but there's no way that I could find to give our container CAP_SYS_RAWIO so the call would actually hit the back door without falling over outside of a vm. 

I'll add a parameter to pass err.